syz-verifier/verifier.go

```diff
@@ -88,19 +88,25 @@ func Verify(res []*Result, prog *prog.Prog, s *Stats) *ResultReport {
 	pool0 := res[0].Pool
 	for _, cr := range rr.Reports {
 		cs := s.Calls[cr.Call]
+
+		mismatch := false
 		for _, state := range cr.States {
 			// For each CallReport verify the ReturnStates from all the pools
 			// that executed the program are the same
 			if state0 := cr.States[pool0]; state0 != state {
 				cr.Mismatch = true
 				send = true
+				mismatch = true
 
-				s.TotalMismatches++
-				cs.Mismatches++
 				cs.States[state] = true
 				cs.States[state0] = true
 			}
 		}
+
+		if mismatch {
```

Comment:
While I think I can guess why you made this change, the commit message merely restating the change is not so useful.

It'd be good to get into the habit of writing commit messages with more background that is not obvious from the code. Usually there was a reason for why you made a change, that reason should be part of the commit message. In this case here, my guess is that the change was made '..., because otherwise the stats would be incorrect with more than 2 VMs." ?

Reply:
Done in PR #2677